bin/cli

```diff
@@ -225,8 +225,13 @@ function warnIfNpmStart (argv, env) {
 
       const projectDirectory = process.argv[4]
 
+      // extract the name from the original package.json
+      const pkgJson = await fs.readJson(path.join(projectDirectory, 'package.json'))
```

Comment:
I think this needs to be done as part of `prepareMigration`, at this point the original `package.json` file will have been blatted...

https://github.com/alphagov/govuk-prototype-kit/blob/c87bf1cca34ce3a3763d94867fb8fe067e0ffa1e/lib/migrator/index.js#L98

Reply:
Done